core/src/main/java/com/rim/IceField/Iceberg.java

```diff
@@ -0,0 +1,127 @@
+package com.rim.IceField;
+
+
+import java.util.ArrayList;
+
+public class Iceberg {
+    private boolean isStable;
+    private int num;
```

Comment:
what does this variable mean? Is it Id, maybe you could find another name that is more verbose? you could use also a static variable to keep track and of the number of objects instantiated and increase it in the constructor.